client/cli/src/client_proxy.rs

```diff
@@ -132,15 +132,12 @@ impl ClientProxy {
         let faucet_account = if faucet_account_file.is_empty() {
             None
         } else {
-            let faucet_account_keypair: KeyPair<Ed25519PrivateKey, Ed25519PublicKey> =
```

Comment:
I think we should keep the keypair and go this way: https://github.com/libra/libra/pull/3414

(only serialize/deserialize the private key pair, generate the public part at loading/creation)

Reply:
I would agree if we ever used the public key, but we don't.